website/docs/docs/dbt-cloud/release-notes/05-Sept-2002/metadata-api-data-retention-limits.md

```diff
@@ -0,0 +1,11 @@
+---
+title: "Query metadata API for data from the previous 3 months"
+id: "metadata-api-data-retention-limits.md"
+description: "Metadata API data retention limits"
+sidebar_label: "Fix: Metadata API data retention limits"
+tags: [Sept-29-2022]
+---
+
+In order to make the metadata API more scalable and improve its latency, we’ve implemented data retention limits. The metadata API can now query data from the previous 3 months. For example, if today was March 1, you could query data back to January 1st.
```

Comment:
I don't think we discussed this as part of the style guide (nor do I necessarily think it needs to be included) but do we want to standardize numbers - should it be '3' or 'three' if it's not something like a date or a phone number?

Reply:
Good call! I think I should write it out as "three" unless it's over 10 or whatever that ding dang rule is :)

Reply:
I made this change and we can follow up with the team to come together with standards